PhysicsTools/PatAlgos/python/producersLayer1/lowPtElectronProducer_cff.py

```diff
@@ -44,14 +44,19 @@
     embedTrack                  = cms.bool(True),
     )
 
+# Rerun IDProducer on pat::Electrons
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronID_cfi import *
+lowPtGsfElectronID.electrons='patLowPtElectrons'
+lowPtGsfElectronID.rho='fixedGridRhoFastjetAll'
```

Comment:
direct modifications of a module used somewhere else is likely going to break things.
clone here or make these additions with some specialized process modifier.
```suggestion
from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronID_cfi import *
patLowPtGsfElectronID = lowPtGsfElectronID.clone( electrons = 'patLowPtElectrons', rho = 'fixedGridRhoFastjetAll')
```

Reply:
BTW, is this intended to always be rerun in miniAOD (even if it's a fresh production starting from RAW/SIM)?

Reply:
Now fixed, pushed to this PR. 

> direct modifications of a module used somewhere else is likely going to break things.
> clone here or make these additions with some specialized process modifier.

Now fixed, pushed to this PR. 

> BTW, is this intended to always be rerun in miniAOD (even if it's a fresh production starting from RAW/SIM)?

For the foreseeable future, yes. We're working on a "deep ID" that may supersede this BDT-based approach. If the study shows significant improvements, we may use e.g. AOD inputs and simply "copy" the new deep ID produced in the RECO process into `electronIDSources`.